src/components/DevPreviewBanner/DevPreviewBanner.tsx

```diff
@@ -37,7 +37,7 @@ export const DevPreviewBanner: FunctionComponent = () => {
         >
           <InfoOutlineIcon h={5} left={4} position="absolute" top={4} w={5} />
           <Text fontSize="md" mx={8} textAlign="center">
-            This application is in preview. Some features may change. If you
+            This application is in Dev Preview. Some features may change. If you
```

Comment:
I don't think capitalization is good here, Dev Preview isn't a known proper noun that users are gonna care about.